src/GuppyDataExplorer/ExplorerTable/index.jsx

```diff
@@ -82,35 +83,48 @@ class ExplorerTable extends React.Component {
     const end = (this.state.currentPage + 1) * this.state.pageSize;
     return (
       <div className={`explorer-table ${this.props.className}`}>
-        <p className='explorer-table__description'>{`Showing ${start} - ${end} of ${totalCount} ${this.props.guppyConfig.dataType}s`}</p>
+        {(this.props.isLocked) ? <React.Fragment />
+          : <p className='explorer-table__description'>{`Showing ${start} - ${end} of ${totalCount} ${this.props.guppyConfig.dataType}s`}</p> }
         <ReactTable
           columns={columnsConfig}
           manual
-          data={this.props.rawData}
-          pages={visiblePages} // Total number of pages, don't show 10000+ records in table
+          data={(this.props.isLocked || !this.props.rawData) ? [] : this.props.rawData}
+          showPageSizeOptions={!this.props.isLocked}
+          // eslint-disable-next-line max-len
+          pages={(this.props.isLocked) ? 0 : visiblePages} // Total number of pages, don't show 10000+ records in table
           loading={this.state.loading}
           onFetchData={this.fetchData}
           defaultPageSize={this.props.defaultPageSize}
           className={'-striped -highlight '}
-          minRows={0} // hide empty rows
+          minRows={3} // make room for no data component
           resizable={false}
+          NoDataComponent={() => (this.props.isLocked ? (
+            <div className='rt-noData'>
```

Comment:
this className looks strange to me. should it be rt-no-data for consistency?

Reply:
ah `rt-noData` is actually defined by `ReactTable`. I'm just overriding its content here

Reply:
aha!! i see